docs/dev/JRS-GettingStarted.md

```diff
@@ -0,0 +1,173 @@
+
+<a name="file-types"></a>
+## **File Types**
+1. _Regression configuration JSON :_ The file name usually starts with `GCP_` (Eg : `GCP-Personal-Neeha-4N.json`,`GCP-Daily-Services-Comp-Basic-4N-1C.json`). Based on the credentials after [GCP setup](#gcp_setup), it holds information of 
+     - `cloud` cloud configuration that includes the private keys, regions in which nodes need to be instantiated etc.,
+     - `experiments` experiments to run.
+     - `slack` slack details needed to post the results. Use `hedera-regression-test` channel for testing. `hedera-regression`/`hedera-regression-summary` channel needs to be used exclusively for nightly regression results.
+     - `result` results folder in the local machine to download the logs after test.  
+     -  `db` postgres information 
+     - other minor details.
+    
+2. _Experiment configuration JSON :_ It holds specific information on the set up of the experiment and validations to be done to tell if it passed.  It majorly includes 
+   - `duration` duration of the test in seconds.
+   - `settings` platform settings that needs to be overridden.
+   - `runConfig` types of the run configuration based on the test type. Eg: `RECONNECT`, `RESTART`, `RECOVER` etc.,
+   - `hederaServicesConfig`  configuration for the clients when running services tests. It includes test suites to be run on client, `CI_PROPERIES_MAP`  to be used, using fixed node for payments etc.,
+   - `app` application jar to be run. Eg: `HederaNode.jar` for services tests.
+   - `startSavedState` saved state location, if any saved state needs to be loaded at the beginning of test.
+   - `validatorConfigs` to be run at end of the experiment to validate if it passed Eg: `ERROR`, `RESTART`, `HEDERA_NODE` etc.,
+
+In order to avoid duplication of data, both JSONs support inheritance from `parentList`. Settings, configurations, validators etc., can be defined in a parent JSON and can be reused as defined in the example below.
+
+```
+"parentList": [
+    "configs/services/default/standard-services-settings.json"
+	"configs/services/default/standard-services-configs.json",
+	"configs/services/default/standard-services-validators.json"
+  ],
+```
+
+**NOTE:** For any developer to run tests, a personal Regression JSON should be created with their personal cloud configuration details.
+It should be created under `swirlds-platform/regression/configs/services/suites/personal/GCP-Personal-XXX.json` (XXX is name of the developer)
+
+**Example of a Personal JSON**
+```
+{
+  "parentList": [
+	"configs/services/JRS-Default-Services.json",
+	"configs/services/GCP-Default-Services-4N.json",
+	"configs/services/suites/daily/JRS-Daily-Default-Services-Slack.json"
+  ],
+  "name": "GCP-Personal-GCPTEST",
```

Comment:
It would be help to mention that the length of the "name" field should be less than the limit. Otherwise, it may not be able to create GCP instances.

Reply:
Agreed @ljianghedera . Will update the doc. Thanks